released/packages/coq-verified-extraction/coq-verified-extraction.0.9.1+8.19/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Matthieu Sozeau <matthieu.sozeau@inria.fr"
+homepage: "https://github.com/yforster/coq-verified-extraction"
+dev-repo: "git+https://github.com/yforster/coq-verified-extraction"
+bug-reports: "https://github.com/yforster/coq-verified-extraction/issues"
+authors:  [
+          "Yannick Forster"
+          "Matthieu Sozeau"
+          "Nicolas Tabareau"
+  ]
+license: "MIT"
+build: [
+  [make]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" { >= "4.13" }
+  "malfunction" { >= "0.6" }
```

Comment:
Do you actually need `malfunction` to build the package or is it only needed when using it? If the latter, instead of depending on it (which means that the package would be recompiled whenever `malfunction` is updated), you could use
```
conflicts: [ "malfunction" { < "0.6" } ]
```

Reply:
We need it during the build, as we bootstrap